Sources/MongoSwift/ChangeStreamEvent.swift

```diff
@@ -1,3 +1,5 @@
+import Foundation
+import SwiftBSON
```

Comment:
nit: can we add a newline after the imports? we usually do that

Reply:
done